### PR TITLE
arch PKGBUILD - depend on xinerama

### PR DESCRIPTION
**Pull request by himdel**
@polyzen I've just updated hsetroot to use libxinerama to determine screen positions and sizes.
(grep, sed and xrandr are no longer needed in path)

Is this change the only thing needed for arch?

### CODE REVIEW

**Comment by himdel** [comment]
(I've also added a `hsr-outputs` executable to list these outputs as text, `--csv`, or `--json`, but not sure whether to install it by default)

File: pkg/arch/PKGBUILD [hunk 1]
```diff
@@ -9,7 +9,7 @@ pkgdesc='A tool which allows you to compose wallpapers for X.  A fork of hsetroo
 arch=('x86_64')
 url=http://github.com/himdel/hsetroot
 license=('GPL2')
-depends=('imlib2' 'xorg-xrandr')
+depends=('imlib2' 'xorg-xrandr','libxinerama')
```
Comment (polyzen) [inline pkg/arch/PKGBUILD]:
```suggestion
depends=('imlib2' 'libxinerama' 'xorg-xrandr')
```

Comment (himdel) [reply pkg/arch/PKGBUILD]: Updated, thanks :)

**Comment by polyzen** [comment]
Looks like #13 isn't a regression, so I will update the repo package after the next release. Thank you :)

**Comment by himdel** [comment]
Perfect, I've just released 1.0.4 (I've also snuck in a color formats fix - https://github.com/himdel/hsetroot/pull/16)

**Comment by polyzen** [comment]
[Done](https://www.archlinux.org/packages/community/x86_64/hsetroot/) 👍

**Pull request merged**